tests/MSA.BuildingBlocks.CosmosDbMigration.Tests/MSA.BuildingBlocks.CosmosDbMigration.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="6.0.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
+    <PackageReference Include="xunit" Version="2.9.2" />
```

Comment:
can we use the latest new reworked xunit v3? https://xunit.net/docs/getting-started/v3/whats-new
https://youtu.be/qIMRFKHldvQ?si=D-9Glh5N1SfbyNJ_